packages/core/src/base-axis-chart.ts

```diff
@@ -136,6 +142,107 @@ export class BaseAxisChart extends BaseChart {
 		}));
 	}
 
+	createSlider(){
+
+		//TODO get the name of the action bar for the chart
+		var divToInsert = "#actions-" + this.options.type
+		console.log(divToInsert)
+
+		var handle_radius = 50,
+			handle_start_val = 50,
+			handle_padding = 10,
+			slider_height = 5,
+			axis_height = 20;
```

Comment:
Any magic numbers and configurations should be added to `Configuration.ts` and loaded in from there